raspbian/applications/vcgencmd.md

```diff
@@ -0,0 +1,131 @@
+## VCGENCMD
+
+`vcgencmd` is a command line utility that can get various pieces of information from the Videocore4 GPU on the Raspberry Pi. Much of the information available is only of use to internal Raspberry Pi developers, but there are number of very useful options available to end users that will be described here.
+
+The source for the application can be found on our github page [here](https://github.com/raspberrypi/userland/tree/master/host_applications/linux/apps/gencmd).
+
+
+### Usage
+
+To get a list of all the commands that `vcgencmd` supports, type `vcgencmd commands`.
+
+Some of the more useful  commands are described below.
+
+### Commands 
+
+#### vcos
+
+The `vcos` cammand has a number of sub commands
+
+`version` Displays the build date and version of the firmware on the VideoCore.
+`log status` Displays the error log status of the various VideoCore software areas.
+
+#### version
+
+Displays the build date and version of the firmware on the VideoCore.
+
+#### get_camera
+
+Displays the enabled  and detected state of the official camera. 1 means yes, 0 means no. Whilst all firmware will support the camera, this support needs to be enabled by using [raspi-config](../../configuration/raspi-config.md).
```

Comment:
"Whilst all firmware will support the camera"

I thought the cutdown firmware didn't support the camera? Or am I getting that confused with something else?

Reply:
You are right, fixed.